SU2_CFD/src/CMarkerProfileReaderFVM.cpp

```diff
@@ -357,9 +357,8 @@ void CMarkerProfileReaderFVM::MergeProfileMarkers() {
       }
     }
 
-    /*--- Sort and remove the duplicate profile marker strings. ---*/
+    /*--- Remove the duplicate profile marker strings. From 1 per point to 1 per marker. ---*/
 
-    sort(profileTags.begin(), profileTags.end());
```

Comment:
so this destroys the order that was always given implicitly by the loop:
```c++
for (unsigned short iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++) {
  if (config->GetMarker_All_KindBC(iMarker) == markerType) {
```
That loop is consequently used during the template writing as well